mailit/tests/plugin_test.py

```diff
@@ -109,11 +109,11 @@ def test_it_only_creates_templates_when_creating_not_when_updating(self):
 class MailSendingTestCase(TestCase):
     def setUp(self):
         super(MailSendingTestCase, self).setUp()
-        self.person3 = Person.objects.all()[2]
+        self.person3 = Person.objects.get(id=1)
```

Comment:
I'm not sure why the variable is called person3, rather than person1, but everything passes for me repeatedly, which is nice, so I'm going to merge.